### ISSUE TITLE
Unexpected exception occurred during `NewHeight()`

### ISSUE THREAD

**Issue by limebell** [planetarium/libplanet#3312]
```[06:17:57 ERR] [ConsensusContext] Unexpected exception occurred during NewHeight()
  System.Collections.Generic.KeyNotFoundException: The given key '-1' was not present in the dictionary.
     at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
     at Libplanet.Net.Consensus.HeightVoteSet.GetVoteSet(Int32 round, VoteFlag voteFlag) in /app/Lib9c/.Libplanet/Libplanet.Net/Consensus/HeightVoteSet.cs:line 222
     at Libplanet.Net.Consensus.HeightVoteSet.PreCommits(Int32 round) in /app/Lib9c/.Libplanet/Libplanet.Net/Consensus/HeightVoteSet.cs:line 178
     at Libplanet.Net.Consensus.Context.GetBlockCommit() in /app/Lib9c/.Libplanet/Libplanet.Net/Consensus/Context.cs:line 245
     at Libplanet.Net.Consensus.ConsensusContext.NewHeight(Int64 height) in /app/Lib9c/.Libplanet/Libplanet.Net/Consensus/ConsensusContext.cs:line 210
     at Libplanet.Net.Consensus.ConsensusContext.<>c__DisplayClass25_0.<<OnTipChanged>b__0>d.MoveNext() in /app/Lib9c/.Libplanet/Libplanet.Net/Consensus/ConsensusContext.cs:line 317```

**Comment by OnedgeLee** [comment]
When received message of higher height and moved to that height with block sync, this may occur, since `ConsensusContext.HandleMessage()` don't trigger `Context.Start()` so `Context.Round` will remain as -1 still.
I'll add round -1 condition on `Context.GetBlockCommit()`, since I think there are no problem generating higher `Context` to receive heigher height message in advance.

But I think there are some attack vector of message spamming via different height, may be this have to be considered later.
(May be this have to be blocked like round spamming)

**Comment by OnedgeLee** [comment]
Already fixed with PR #3276, closing.
